SwiftMessages/CALayer+Utils.swift

```diff
@@ -6,7 +6,7 @@
 //  Copyright © 2018 SwiftKick Mobile. All rights reserved.
 //
 
-import CoreGraphics
+import QuartzCore
```

Comment:
Is this necessary or is this just a drive-by fix?

Reply:
This is required for SPM, some CALayer-stuff won't build with just CoreGraphics imported